src/data/systemIntake.ts

```diff
@@ -254,3 +254,41 @@ export const convertIntakeToCSV = (intake: SystemIntakeForm) => {
     archivedAt: intake.archivedAt && intake.archivedAt.toISO()
   };
 };
+
+/**
+ * Check if any intake fields have been filled out
+ */
+export const isIntakeStarted = (intake: SystemIntakeForm) => {
```

Comment:
I'm open to ideas to better to this. I was thinking some sort of recursion that can read nested objects, but it got too complicated since we need to omit certain attributes and also check arrays.

This is brittle since we _should_ update it every time we add a field. However it probably only really matters that we check the first page of fields.

Reply:
The only other thing I could think to check would be a difference between the created_at and the updated_at. If those two are the same, then we haven't done any edits since original create. If they are different, we have. BUT I am ok with what you have here, and agree that checking only the first page of info is a good move.

Reply:
Ohhhhhh that's a good idea. Hmmm, let me think about that.